backend/test/conftest.py

```diff
@@ -207,7 +207,7 @@ def _crawler_create_config_only(crawler_auth_headers, default_org_id):
     # Start crawl.
     crawl_data = {
         "runNow": False,
-        "name": "Crawler User Test Crawl",
+        "name": "crawler User Test Crawl",
```

Comment:
This change is to show that 'c' gets sorted next to 'C' in some of the tests, right?

Reply:
Yeah it's to show that it gets sorted where a word starting with `c` should compared to the other crawls, regardless of casing.